source/PlayniteSDK/Plugins/IGameLibrary.cs

```diff
@@ -29,6 +29,12 @@ public interface ILibraryPlugin : IPlugin
         /// </summary>
         string Name { get; }
 
+        /// <summary>
+        /// Gets if this game library provider is installed on PC
+        /// </summary>
+        /// <returns></returns>
+        bool IsInstalled { get; }

```

Comment:
Please rename to something like `IsClientInstalled`, I think it would better represent what the property should be used for.